app/scripts/router.js

```diff
@@ -137,10 +137,8 @@ function (
       this.formPrefill = options.formPrefill;
       this.notifications = options.notifications;
       this.able = options.able;
+      this.storage = Storage.factory('sessionStorage', this.window);
```

Comment:
Not for this PR because has been a problem for 2 years - the mixture of private vars with the `_` prefix and those without causes me grief.